validator/client/validator_test.go

```diff
@@ -1984,78 +1983,6 @@ func TestValidator_PushProposerSettings(t *testing.T) {
 			},
 			err: "could not submit signed registrations to beacon node",
 		},
-		{
-			name: "Validator Index Not found with validator registration",
```

Comment:
This test is outdated. The validator index is irrelevant for validator registration. Validator registration doesn't use the validator index. It only uses public key